content/javascript/concepts/strings/terms/valueOf/valueOf.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: '.valueOf()'
+Description: 'Returns the value of a String object as a string.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Objects'
+  - 'Strings'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.valueOf()`** method returns the [primitive](https://www.codecademy.com/resources/docs/javascript/data-types) value of a `String` object, similar to the `.toString()` method of JavaScript Strings. Usually this method isn't used in code but is called internally by JavaScript.
```

Comment:
Thanks for looking into common use case of this method! Since we have this last sentence here, could we rephrase the preface to the code byte below? (see below)

```suggestion
The **`.valueOf()`** method returns the [primitive](https://www.codecademy.com/resources/docs/javascript/data-types) value of a `String` object, similar to the `.toString()` method of JavaScript Strings. Usually this method is called internally by JavaScript.
```